peer_mentoring/groups/views.py

```diff
@@ -126,7 +126,7 @@ def send_group_join_request(request):
 
 def group_request_index(request, group_id):
     group = get_object_or_404(Group, id=group_id)
-    requests = GroupJoinRequest.objects.filter(id=group_id)
+    requests = GroupJoinRequest.objects.filter(group=group)
```

Comment:
good catch, id=group_id did not seem right, because you're matching different ids, I also use the object when I filter on related field

Reply:
Thanks, it was a bit of TDD I suppose, the test was catching the bug and I changed it.

Reply:
that's awesome!